### ISSUE TITLE
getAnnotationsOnPage.xql could return "wrong" data.

### ISSUE THREAD

**Issue by nikobeer** [Edirom/Edirom-Online#83]
It should be possible to exactly specify where annotations are gathered from. As I understand annotations are searched everywhere in '/db/contents/'? So if you have multiple MEI files with annotations you will possibly get "wrong" annotations (see l. 78).

Example from OPERA: working with two MEI files for prerelease final correction -> only "the first" (archived) MEI file is taken into account?

Further problem: This seems to be completely independent from getting annotations for the Annotations List and View? Here the correct file is found (specified). As a consequence you could get different annotations in annotation view compared to source view.

Would it be possible to centralize annotation management within Edirom?

**Comment by bwbohl** [comment]
this is true, while the annotation list is populated from the `work.mei` the getAnnotationsOnPage searches for all `mei:annot` in `prefs:edition_path`

**Comment by bwbohl** [comment]
I would suggest to define corresponding `getAnnotations` functions in the annotation module.

**Comment by bwbohl** [comment]
this seems connected to #156 and #240

**Comment by bwbohl** [comment]
@hizclick yet another one for the collection?

**Comment by hizclick** [comment]
@nikobeer, is this issue already resolved or how can we reproduce the issue?
